hack/generate/samples/internal/go/v3/memcached_with_webhooks.go

```diff
@@ -369,6 +427,8 @@ const reconcileFragment = `// Fetch the Memcached instance
 			log.Error(err, "Failed to update Deployment", "Deployment.Namespace", found.Namespace, "Deployment.Name", found.Name)
 			return ctrl.Result{}, err
 		}
+		// Increment memcachedNumberOfReplicasReconcileCountTotal metric by 1
+		metrics.MemcachedNumberOfReplicasReconcileCountTotal.Inc()
```

Comment:
I would prefer not to export the `MemcachedNumberOfReplicasReconcileCountTotal` in `metrics.go`, but instead create an exported helper function just to increment the metric
`metrics.IncrementMemcachedNumberOfReplicasReconcileCount()`

Reply:
Did you mean export `MemcachedNumberOfReplicasReconcileCountTotal` to `memcached_controller.go`?
If so, can you please explain the profit of your suggestion?
If the operator will have a few counter metrics, we will need to implement many different functions that are all responsible to `Inc()` a metric. In addition, your suggestion still requires importing `metrics` package to `memcached_controller.go`.